tests/db_access/prepared_statements.cpp

```diff
@@ -567,6 +568,48 @@ go_bandit([](){
             {
             }
         });
+
+        it("can work with prepared statement helper functions - psReadValues", [&](){
+            {
+                int id{};
+                BlobData blob{}, binary{}, varbinary{};
+                psReadValues("SELECT `id`, `blob`, `binary`, `varbinary` FROM `test_superior_sqlpp`.`binary_data` ORDER BY `id` LIMIT 1", connection, id, blob, binary, varbinary);
+
+                AssertThat(id, Equals(42));
+                AssertThat(blob.size(), Equals(5u));
+                AssertThat(binary.size(), Equals(10u));
+                AssertThat(varbinary.size(), Equals(5u));
+                AssertThat(blob.getStringView()=="ab0cd"s, IsTrue());
+                AssertThat(binary.getStringView(), Equals("ef\0gh\0\0\0\0\0"s));
+                AssertThat(varbinary.getStringView(), Equals("ij\0kl"s));
+            }
+            {
+                int id{};
+                BlobData blob{}, binary{}, varbinary{};
+                AssertThrows(UnexpectedMultipleRowsError,
```

Comment:
I don't get it, why this should throw? The order and types of arguments seems to be correct?